src/indexers/yield-estimates-indexer.ts

```diff
@@ -1,150 +1,174 @@
-          lastMeasuredAt: existingHarvest.lastMeasuredAt ?? 0,
-          duration: existingHarvest.duration ?? Number.MAX_SAFE_INTEGER,
-          lastReportedAt: existingHarvest.lastReportedAt ?? 0,
-        };
-
-        let shouldCheckGraph = false;
-        const now = Date.now();
-
-        if (vault.version && vault.version === VaultVersion.v1_5) {
-          try {
-            const pendingHarvest = await sdk.vaults.getPendingHarvest(vault.address);
-
-            harvestData.harvestTokens = await Promise.all(
-              pendingHarvest.tokenRewards.map(async (t) => toBalance(await getFullToken(chain, t.address), t.balance)),
-            );
-
-            harvestData.lastHarvestedAt = pendingHarvest.lastHarvestedAt * 1000;
-          } catch {
-            shouldCheckGraph = true;
-            if (now - ONE_DAY_MS > harvestData.lastReportedAt) {
-              // TODO: enable once its not busted
-              // sendPlainTextToDiscord(
-              //   `${chain.network} ${vault.name} (${vault.protocol}, ${vault.version}, ${
-              //     vault.state ?? VaultState.Open
-              //   }) failed to harvest!`,
-              // );
-              harvestData.lastReportedAt = now;
-            }
-          }
+async function loadGraphTimestamp(sdk: BadgerSDK, vault: VaultDefinitionModel): Promise<number> {
+  let lastHarvestedAt = 0;
+
+  const { settHarvests } = await sdk.graph.loadSettHarvests({
+    first: 1,
+    where: {
+      sett: vault.address.toLowerCase(),
+    },
+    orderBy: SettHarvest_OrderBy.Timestamp,
+    orderDirection: OrderDirection.Desc,
+  });
+  const { badgerTreeDistributions } = await sdk.graph.loadBadgerTreeDistributions({
+    first: 1,
+    where: {
+      sett: vault.address.toLowerCase(),
+    },
+    orderBy: BadgerTreeDistribution_OrderBy.Timestamp,
+    orderDirection: OrderDirection.Desc,
+  });
+
+  if (settHarvests.length > 0) {
+    lastHarvestedAt = settHarvests[0].timestamp * 1000;
+  }
+  if (badgerTreeDistributions.length > 0 && badgerTreeDistributions[0].timestamp > lastHarvestedAt) {
+    lastHarvestedAt = badgerTreeDistributions[0].timestamp * 1000;
+  }
+
+  return lastHarvestedAt;
+}
 
-          const pendingYield = await sdk.vaults.getPendingYield(vault.address);
-          harvestData.yieldTokens = await Promise.all(
-            pendingYield.tokenRewards.map(async (t) => toBalance(await getFullToken(chain, t.address), t.balance)),
-          );
-        } else {
-          shouldCheckGraph = true;
-        }
+export async function refreshYieldEstimates() {
+  const chains = getSupportedChains();
+  const now = Date.now();
+
+  for (const chain of chains) {
+    const convert = async (t: TokenBalance) => toTokenValue(chain, t);
+    const sdk = await chain.getSdk();
+    const mapper = getDataMapper();
+    const vaults = await chain.vaults.all();
+
+    for (const vault of vaults) {
+      if (vault.state === VaultState.Discontinued || vault.protocol !== Protocol.Aura) {
```

Comment:
so only aura vaults get yield estimates?

Reply:
resolved, was just for local testing